compiler-plugin/src/main/java/io/ballerina/lib/np/compilerplugin/PromptAsCodeCodeModificationTask.java

```diff
@@ -110,7 +110,7 @@ public class PromptAsCodeCodeModificationTask implements ModifierTask<SourceModi
             NodeFactory.createSimpleNameReferenceNode(NodeFactory.createIdentifierToken(CONTEXT_VAR));
 
     private final ModifierData modifierData;
-    private final CodeModifier.AnalysisData analysisData;
+    private static CodeModifier.AnalysisData analysisData;
```

Comment:
Why was this changed? Why can't this be used similar to modifierData? 

Note the warnings also - Static member 'io.ballerina.lib.np.compilerplugin.PromptAsCodeCodeModificationTask.analysisData' accessed via instance reference